wandb/plot/confusion_matrix.py

```diff
@@ -4,13 +4,14 @@
 chart_limit = wandb.Table.MAX_ROWS
 
 
-def confusion_matrix(preds=None, y_true=None, class_names=None):
+def confusion_matrix(probs=None, y_true=None, preds=None, class_names=None):
     """
     Computes a multi-run confusion matrix.
 
     Arguments:
-        preds (arr): Array of predicted label indices.
+        probs (2-d arr): Shape [n_examples, n_classes]
```

Comment:
This is great! Can we add a usage example or two in the description, perhaps one with preds and one with probs? I would also include a bit more descriptive text because "probs" and "preds" are so similar looking and easy to confuse.

Reply:
Added one example, I could argmax it for the second, thoughts?

Reply:
I think this is great!